osCam/userconfig/models.py

```diff
@@ -1,9 +1,9 @@
 from django.db import models
-from django.contrib.auth.models import User
 from django.contrib.auth.models import AbstractUser
 
+
 class CustomUser(AbstractUser):
-     emailKey = models.CharField(max_length=16, blank=True)
+    emailKey = models.CharField(max_length=16, blank=True)
```

Comment:
to future proof, we may want to make the max length longer, but maybe not worth thinking about right now.